### PR TITLE
added isort support

### PR DESCRIPTION
**Pull request by dineshkrishnareddy**
Added isort support for repo https://github.com/census-instrumentation/opencensus-python/issues/478

### CODE REVIEW

**Comment by dineshkrishnareddy** [comment]
@googlebot I signed it!

File: docs/trace/usage.rst [hunk 1]
```diff
@@ -1,25 +1,42 @@
...
 
+--------
+ Tracing
+--------
+
 Installation & basic usage
 --------------------------
 
-1. Install the opencensus-trace package using `pip`_ or `pipenv`_:
+1. Install the opencensus package using `pip`_ or `pipenv`_:
```
Comment (lzchen) [inline docs/trace/usage.rst]: I don't think we want to change the content of the docs.

**Comment by lzchen** [comment]
@dineshkrishnareddy 
Thanks so much for contributing! The PR looks good and thanks for following what we have in OpenTelemetry. I would recommend pulling the latest `master` branch` as there are some files that are missing such as [#793]. I left a comment. Also please fix your CI :).

**Comment by dineshkrishnareddy** [comment]
@lzchen thanks, fixed CI. Please have a look

**Pull request merged**